mteb/tasks/Clustering/multilingual/MLSUMClusteringP2P.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+import datasets
+import numpy as np
+
+from mteb.abstasks import AbsTaskClustering, MultilingualTask, TaskMetadata
+
+_LANGUAGES = {
+    "de": ["deu-Latn"],
+    "fr": ["fra-Latn"],
+    "ru": ["rus-Cyrl"],
+    "es": ["spa-Latn"],
+}
+# Did not include turkish (tu) samples because all `topics` values are set to "unknown".
+# Which results in a v-measure of 1 as all texts are considered to be in one cluster.
+
+
+class MLSUMClusteringP2P(AbsTaskClustering, MultilingualTask):
+    metadata = TaskMetadata(
+        name="MLSUMClusteringP2P",
+        description="Clustering of newspaper article contents and titles from MLSUM dataset. Clustering of 10 sets on the newpaper article topics.",
+        reference="https://huggingface.co/datasets/mlsum",
+        dataset={
+            "path": "mlsum",
+            "revision": "b5d54f8f3b61ae17845046286940f03c6bc79bc7",
+            "trust_remote_code": True,
+        },
+        type="Clustering",
+        category="p2p",
+        eval_splits=["validation", "test"],
+        eval_langs=_LANGUAGES,
+        main_score="v_measure",
+        date=("2010-01-01", "2018-09-30"),
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
+        license="Not specified",
+        socioeconomic_status="mixed",
+        annotations_creators="found",
```

Comment:
Seems like tests are failing here. Same below.
```suggestion
        annotations_creators="derived",
```